src/harness/testcases/testdata/sas_impl_record_0.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "sas_impl/0/0",
+  "name": "Sas Implementation Record",
+  "administratorId": "sas_admin/sas/0",
```

Comment:
Updated in SAS-SAS TS: I think "sas_admin/test_admin_name" would be correct given the new spec.